site/style.css

```diff
@@ -1,11 +1,137 @@
+body {
+  margin:0;
+  }
+
+h1 {
+  font-family: 'Oswald', sans-serif;
+  font-size: 60px;
+  color: #FFF;
+  }
+
+h2 {
+  font-family: 'Oswald', sans-serif;
+  font-size: 40px;
+  color: #020D38;
+  }
+
+h3 {
+  font-family: 'Lato', sans-serif;
+  font-size: 30px;
+  color: #020D38;
+  }
+
+h4 {
+  font-family: 'Lato', sans-serif;
+  font-size: 18px;
+  color: #06259E;
+  }
+
 p {
-	color: red;
-}
+  font-family: 'Lato', sans-serif;
+  font-size: 18px;
+  line-height: 1.5;
+  }
+
+a {
+  font-family: 'Lato', sans-serif;
+  font-size: 18px;
+  color: #FFF;
+  }
 
-.greenparagraph {
-	color: blue;
+tr, td {
+  padding: 10px;
 }
 
-p span {
-	color: #831FB5;
-}
\ No newline at end of file
+li {
+  font-family: 'Lato', sans-serif;
+  font-size: 18px;
+  }
+
+li a {
+  display: block;
+  color: white;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  }
+
+li a:hover:not(.active) {
```

Comment:
Nice use of :not here!  It is not used all that often, but really useful to know about